subprojects/maven/src/integTest/groovy/org/gradle/api/publish/maven/MavenPublishVersionRangeIntegTest.groovy

```diff
@@ -16,15 +16,20 @@
 
 package org.gradle.api.publish.maven
 
-import org.gradle.integtests.fixtures.ToBeFixedForConfigurationCache
 import org.gradle.integtests.fixtures.publish.maven.AbstractMavenPublishIntegTest
 
 class MavenPublishVersionRangeIntegTest extends AbstractMavenPublishIntegTest {
     def mavenModule = javaLibrary(mavenRepo.module("org.gradle.test", "publishTest", "1.9"))
 
-    @ToBeFixedForConfigurationCache
     void "version range is mapped to maven syntax in published pom file"() {
         given:
+
+        javaLibrary(mavenRepo.module("group", "projectA", "1.0")).withModuleMetadata().publish()
```

Comment:
💄 Extract method `publishJavaLibrary(String group, String artifact, String version)`

```groovy
    publishJavaLibrary("group", "projectA", "1.0")
    publishJavaLibrary("group", "projectB", "1.0")
    ...
```

Perhaps better:
```groovy
        for (char suffix : 'ABCDE') {
            publishJavaLibrary("group", "project$suffix", "1.0")
        }
```

It might make sense to move the method up in the hierarchy to simplify the other tests.

Reply:
Applied the looping suggestion. The `publishJavaLibrary` helper would cover one common variation, but not the long tail of less common variations (publishing platforms vs java libraries vs..., with/without metadata, etc), which the MavenJavaModule DSL expressivity handles well. The use of a loop addresses the repetition well though.